src/pages/library/DataBrowserPreview.js

```diff
@@ -139,9 +152,14 @@ const DataBrowserPreview = ({ id }) => {
             isSearchable: true,
             isClearable: false,
             value: selectedTable,
-            getOptionLabel: ({ value }) => `${_.startCase(value)}${tableMap[value].rowCount === 0 ? ' (0 Rows)' : ''}`,
+            getOptionLabel: ({ rowCount, value }) => {
+              return div({ style: { color: colors.dark(!!rowCount ? 1 : 0.5) } }, [_.startCase(value)])
+            },
+            formatGroupLabel: ({ label }) => {
+              return div({ style: { marginTop: 5, paddingTop: 15, borderTop: `1px solid ${colors.dark(0.5)}`, color: colors.dark(0.8) } }, [label])
+            },
```

Comment:
It doesn't feel like the vertical spacing helps here. Consider:

```suggestion
            getOptionLabel: ({ rowCount, value }) => div({ style: { color: colors.dark(!!rowCount ? 1 : 0.5) } }, [_.startCase(value)]),
            formatGroupLabel: ({ label }) => div({ style: { marginTop: 5, paddingTop: 15, borderTop: `1px solid ${colors.dark(0.5)}`, color: colors.dark(0.8) } }, [label]),
```

Reply:
I thought the line was just really long in the ide, and would require (for my monitor at least) some side scrolling to read the whole line. I'd rather keep it as is

Reply:
I'm fine with merging as is but so you know, there's sort of a best of both worlds if you're using an IDE like IntelliJ, by turning on [soft wrapping](https://www.jetbrains.com/idea/guide/tips/enable-soft-wrap). Notice how a long line like below is visually split with diagonal arrows showing that there's soft wrapping, without the author having to actually split the lines.

![image](https://user-images.githubusercontent.com/5438223/146933309-a7262496-4449-435e-88ee-ca4cc7701656.png)